src/test/java/org/fisco/bcos/channel/client/CallContractTest.java

```diff
@@ -0,0 +1,303 @@
+import org.fisco.bcos.web3j.abi.datatypes.Utf8String;
+import org.fisco.bcos.web3j.abi.datatypes.generated.Int16;
+import org.fisco.bcos.web3j.abi.datatypes.generated.Int256;
+import org.fisco.bcos.web3j.abi.datatypes.generated.StaticArray2;
+import org.fisco.bcos.web3j.abi.datatypes.generated.Uint16;
+import org.fisco.bcos.web3j.crypto.Credentials;
+import org.fisco.bcos.web3j.crypto.ECKeyPair;
+import org.fisco.bcos.web3j.crypto.gm.GenCredential;
+import org.fisco.bcos.web3j.protocol.Web3j;
+import org.fisco.bcos.web3j.protocol.channel.ChannelEthereumService;
+import org.fisco.bcos.web3j.protocol.core.methods.response.TransactionReceipt;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+public class CallContractTest {
+    static Logger logger = LoggerFactory.getLogger(CallContractTest.class);
+    public static Web3j web3j;
+
+    public static ECKeyPair keyPair;
+    public static Credentials credentials;
+    public static BigInteger gasPrice = new BigInteger("3000000000");
+    public static BigInteger gasLimit = new BigInteger("3000000000");
+
+    public static void main(String[] args) throws Exception {
+
+        try {
+            // init the Service
+            ApplicationContext context =
+                    new ClassPathXmlApplicationContext("classpath:applicationContext.xml");
+
+            Service service = context.getBean(Service.class);
+            service.setGroupId(Integer.parseInt(args[0]));
+            service.run(); // run the daemon service
+            // init the client keys
+            credentials = GenCredential.create();
+
+            logger.info("-----> start test in CallContractTest!");
+            ChannelEthereumService channelEthereumService = new ChannelEthereumService();
+            channelEthereumService.setTimeout(10 * 1000);
+            channelEthereumService.setChannelService(service);
+            web3j = Web3j.build(channelEthereumService, Integer.parseInt(args[0]));
+
+            if (args.length == 2) {
+                String address = args[1];
+                CallContract callContract = new CallContract(credentials, web3j);
+                System.out.println(
+                        "************************ Test call & sendTrandation ************************");
+                testSyncCallContract(callContract, address);
+                testAsyncCallContract(callContract, address);
+
+                System.out.println("************************ Test decode ************************");
+                testDecode(callContract, address);
+                System.out.println("Test CallContract successfully.");
+            } else {
+                System.out.println("Please input group id and contract address.");
+            }
+        } catch (Exception e) {
+            System.out.println(e.getLocalizedMessage());
+            System.exit(1);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [System.exit() should not be used in J2EE/JEE apps](https://app.codacy.com/app/fisco/web3sdk/pullRequest?prid=4732380)